network/src/main/java/com/tapptitude/network/source/ImageRemoteDataSource.kt

```diff
@@ -0,0 +1,14 @@
+package com.tapptitude.network.source
+
+import com.tapptitude.network.api.ImageApi
+import com.tapptitude.network.dto.ImageDto
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.withContext
+
+class ImageRemoteDataSource(private val imageApi: ImageApi) {
+    suspend fun getRandomImage(): ImageDto {
+        return withContext(Dispatchers.IO) {
```

Comment:
we could get rid of the return statement and rewrite it as
`suspend fun getRandomImage(): ImageDto = withContext(Dispatchers.IO) {`
`imageApi.getRandomImage()`
`}`